src/OrchardCore.Modules/OrchardCore.Users/Recipes/UsersRecipeStep.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Recipes.Services;
+using OrchardCore.Security;
+using OrchardCore.Security.Services;
+using OrchardCore.Users.Models;
+using OrchardCore.Users.Services;
+
+namespace OrchardCore.Users.Recipes
+{
+    public class UsersRecipeStep : IRecipeStepHandler
```

Comment:
I think you should remove this step from the pr.

I think the pr would pretty much be ready to go, if this was dropped.

Few reasons
- security concerns about plain text passwords in json files
- already have user commands to do this
- can probably resolve / question the security concerns again with the secrets pr, as there would at least be a way to encrypt them.

Reply:
Sorry, It is not clear to me either, why do we need UsersRecipeStep? As I know SetupService should create the admin user.